### ISSUE TITLE
Incompatible Plug-ins Disabled

### ISSUE THREAD

**Issue by piratx** [Free-GPGMail/Free-GPGMail#29]
Hello, thank you for this plugin.
I am on Monterey 12.1 but I am always getting this error when I enable the plugin and restart Mail. Is there something I am missing here?
<img width="372" alt="Screen Shot 2021-12-10 at 8 54 29 AM" src="https://user-images.githubusercontent.com/242366/145530660-43d7364e-c669-4164-aad8-922ac57cd95e.png">

Thanks in advance

**Comment by bnavigator** [comment]
Hi,

Could you please try if
[Free-GPGMail_6_12.1.mailbundle.zip](https://github.com/Free-GPGMail/Free-GPGMail/files/7691734/Free-GPGMail_6_12.1.mailbundle.zip)
works for you?

If not, could you please tell me the output of these two commands?
```
sw_vers -productVersion
defaults read /System/Applications/Mail.app/Contents/Info PluginCompatibilityUUID
```

Thanks.

**Comment by piratx** [comment]
12.1
6FF8B077-81FA-45A4-BD57-17CDE79F13A5

I have tried it, it doesnt work :( Thank in advance!

**Comment by bnavigator** [comment]
Thanks, that's a new UUID. I wonder whether we need to do this dance for every minor Monterey update :(

Please try again:
[Free-GPGMail_6_12.1_2.mailbundle.zip](https://github.com/Free-GPGMail/Free-GPGMail/files/7691934/Free-GPGMail_6_12.1_2.mailbundle.zip)

**Comment by piratx** [comment]
It works great now! Thank you!

> I wonder whether we need to do this dance for every minor Monterey update :(

Let's hope not.. :(

**Comment by bnavigator** [comment]
Fixed in new release v2021.2p3
